tests/test_commands.py

```
@@ -107,16 +107,49 @@ def test_dianncfg_example(self):
         assert result.exit_code == 0
 
+    def test_check_samplesheet_minimal_missing_column(self):
+        """Test minimal validation fails when a required column is missing."""
+        import tempfile
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".sdrf.tsv", delete=False) as f:
+            # Missing comment[cleavage agent details]
+            f.write("source name\tassay name\tcomment[data file]\tcomment[label]\t"
+                    "comment[instrument]\tcomment[proteomics data acquisition method]\t"
+                    "technology type\tcomment[modification parameters]\n")
+            f.write("S1\trun1\tfile1.raw\tlabel free sample\tOrbitrap\tDIA\tMS\tOxidation\n")
+            f.name
```

Ruff (0.15.6)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)